src/components/form/edit/attachments.vue

```diff
@@ -33,19 +34,47 @@ except according to the terms contained in the LICENSE file.
 
 <script setup>
 import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
 
 import FormAttachmentList from '../../form-attachment/list.vue';
 import FormEditSection from './section.vue';
 import Loading from '../../loading.vue';
 
+import { useI18nUtils } from '../../../util/i18n';
 import { useRequestData } from '../../../request-data';
 
-const { draftAttachments } = useRequestData();
+const { form, publishedAttachments, draftAttachments, resourceStates } = useRequestData();
+const { dataExists } = resourceStates([form, publishedAttachments, draftAttachments]);
 
 const hasAttachments = computed(() =>
   draftAttachments.dataExists && draftAttachments.size !== 0);
 const hasMissing = computed(() =>
   draftAttachments.dataExists && draftAttachments.missingCount !== 0);
+
+const { t } = useI18n();
+const { tn, formatList } = useI18nUtils();
+const tag = computed(() => {
+  if (!(dataExists.value && hasAttachments.value && form.publishedAt != null))
```

Comment:
We don't need to check `form.publishedAt != null` explicitly because there won't be `publishedAttachments` if the form isn't published.